builder.config.json

```diff
@@ -346,7 +351,13 @@
       "SMS",
       "USSD",
       "IVR"
-    ]
+    ],
+    "appWideInteractionTimeout": null,
+    "title": {
+      "createFlow": "flow-builder.create-flow",
+      "editFlow": "flow-builder.edit-flow",
+      "importFlow": "flow-builder.import-flow"
+    }
```

Comment:
I'm glad you undid my `showValidationErrorsOnlyAfterAttemptToPublishTree` config.
It was ugly. And it added unnecessary complexity.
Now both the community version and the embedded app will have the same behaviour (the errors will be shown only after the user tries to publish). And that's fine (I was trying to make it configurable, but no one said that it should be in the first place). Thanks!

Reply:
I also realize now that the AC was talking about validation errors on the `NewFlow` window and the flow creation action as a trigger for them. (And _not_ about the flow/block validation errors in the interaction designer and the publishing action as a trigger).

Reply:
No worries, sorry if there were some missed explanation from the ticket. Happy we made this together ;)